Fix Selenium UI tests to work with new jersey

In Gaffer 2.0.0-alpha-3, jersey was updated to 2.36 ([Gaffer #2689](https://github.com/gchq/Gaffer/pull/2689)). This involved some dependency changes. These have caused the Selenium UI tests to fail with:
```
[ERROR] uk.gov.gchq.gaffer.ui.QueryBuilderST  Time elapsed: 3.563 s  <<< ERROR!
java.lang.IllegalStateException: InjectionManagerFactory not found.
	at org.glassfish.jersey.internal.inject.Injections.lambda$lookupInjectionManagerFactory$0(Injections.java:74)
	at java.base/java.util.Optional.orElseThrow(Optional.java:408)
	at org.glassfish.jersey.internal.inject.Injections.lookupInjectionManagerFactory(Injections.java:74)
	at org.glassfish.jersey.internal.inject.Injections.createInjectionManager(Injections.java:44)
	at org.glassfish.jersey.client.ClientConfig$State.initRuntime(ClientConfig.java:413)
	at org.glassfish.jersey.internal.util.collection.Values$LazyValueImpl.get(Values.java:317)
	at org.glassfish.jersey.client.ClientConfig.getRuntime(ClientConfig.java:820)
	at org.glassfish.jersey.client.ClientRequest.getClientRuntime(ClientRequest.java:176)
	at org.glassfish.jersey.client.ClientRequest.getInjectionManager(ClientRequest.java:567)
	at org.glassfish.jersey.client.JerseyWebTarget.onBuilder(JerseyWebTarget.java:371)
	at org.glassfish.jersey.client.JerseyWebTarget.request(JerseyWebTarget.java:199)
	at org.glassfish.jersey.client.JerseyWebTarget.request(JerseyWebTarget.java:38)
	at uk.gov.gchq.gaffer.proxystore.ProxyStore.createRequest(ProxyStore.java:272)
	at uk.gov.gchq.gaffer.proxystore.ProxyStore.doGet(ProxyStore.java:232)
```

Making similar jersey dependency changes to Gaffer should get the tests working again.

--- a/ui/pom.xml
+++ b/ui/pom.xml
@@ -48,6 +48,11 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.glassfish.jersey.inject</groupId>
+            <artifactId>jersey-hk2</artifactId>
+            <version>${jersey.version}</version>
+        </dependency>
+        <dependency>
             <groupId>uk.gov.gchq.gaffer</groupId>
             <artifactId>proxy-store</artifactId>
             <version>${gaffer.version}</version>

--- a/ui/pom.xml
+++ b/ui/pom.xml
@@ -43,8 +43,8 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <!-- Newer Guava required for Selenium -->
-            <version>25.0-jre</version>
+            <!-- Guava >= 25 required for Selenium -->
+            <version>[25.0-jre,)</version>
             <scope>test</scope>
         </dependency>
         <dependency>

